.github/wip-labeler.yml

```diff
@@ -0,0 +1,6 @@
+# Add 'wip' label to all new PRs
+wip:
```

Comment:
This could be confusing if people aren't expecting `WIP` to be added by default.

Reply:
I suppose so, but it shouldn't take long to get used to.